src/Sulu/Bundle/SecurityBundle/Entity/BaseRole.php

```diff
@@ -44,6 +44,7 @@
 
     public function __construct()
     {
+        parent::__construct('');
```

Comment:
Does it make sense to override the constructor at all?

Reply:
required property of the [symfony constructor](https://github.com/symfony/symfony/blob/8d277ce3e594c196594170e4fbbc37be98ba4f73/src/Symfony/Component/Security/Core/Role/Role.php#L23).
but could move the parent call to the Role class?

Reply:
No, in that case it is fine I suppose.